content/en/docs/reference/configuration/jwk/_index.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "JSON Web Keys (JWK)"
+linkTitle: "JSON Web Keys (JWK)"
+weight: 7
+---
+  
+  
+JSON Web Key (JWK) is a JSON data structure that represents a single public key or a set of
+public keys. IAM uses private JWKs to sign tokens, and IAM clients use public JWKs to validate the tokens.
+A [JWK Generator][jwk-generator] can be used to generate new keys.
+
+### Keystore Location
+
+JWKs are stored in a file that is called keystore. All the keys in this file will be 
+used for validating the tokens. Keystore location is set using the `IAM_KEY_STORE_LOCATION` 
+environment variable, e.g.:
+
+```env
+IAM_KEY_STORE_LOCATION=file:///secrets/jwk/keystore.jwks
+```
+
+
+### Default Key ID (kid)
+
+If there are multiple JWKs in the keystore, `IAM_JWK_DEFAULT_KEY` environment variable
+can be set to choose the the key that signs the tokens, e.g.:
```

Comment:
Typo: the the key -> the key

I'd add "`IAM_JWK_DEFAULT_KEY` environment variable can be set to choose the key that signs the tokens by specifying the _kid_, e.g.:"